ublox/src/ubx_packets/packets/mga_bds_eph.rs

```diff
@@ -0,0 +1,124 @@
+#[cfg(feature = "serde")]
+use super::SerializeUbxPacketFields;
+#[cfg(feature = "serde")]
+use crate::serde::ser::SerializeMap;
+
+use crate::{error::ParserError, UbxPacketMeta};
+use ublox_derive::ubx_packet_recv;
+
+/// UBX-MGA-BDS Ephemeris packet.
+#[ubx_packet_recv]
+#[ubx(class = 0x13, id = 0x03, fixed_payload_len = 88)]
+struct MgaBdsEph {
+    /// Message type. 0x01 for this type.
+    msg_type: u8,
+
+    /// Message version.
+    version: u8,
+
+    /// BeiDou Satellite Identifier.
+    sv_id: u8,
+
+    /// Reserved
+    reserved1: u8,
+
+    /// Autonomous Satellite Health H1 flag
+    sat_h1: u8,
+
+    /// Issue of Data, Clock
+    iodc: u8,
+
+    /// a2 correction term (in seconds.s⁻²)
```

Comment:
The units for this field in the docstring is incorrect, right? It is different then in the other fields, should we make it consistent ? (seconds / seconds^2) Or something similar?